kroxylicious-systemtests/src/main/java/io/kroxylicious/systemtests/logs/TestLogCollector.java

```diff
@@ -0,0 +1,232 @@
+ *     │              │          │          └── strimzi-cluster-operator.yaml
+ *     │              │          ├── kafka
+ *     │              │          │          └── my-cluster.yaml
+ *     │              │          ├── kafkanodepool
+ *     │              │          │          └── kafka.yaml
+ *     │              │          ├── events.log
+ *     │              │          ├── pod
+ *     │              │          │          ├── describe-pod-my-cluster-kafka-0.log
+ *     │              │          │          ├── describe-pod-my-cluster-kafka-1.log
+ *     │              │          │          ├── describe-pod-my-cluster-kafka-2.log
+ *     │              │          │          └── logs-pod-strimzi-cluster-operator-57d455495c-98t8f-container-strimzi-cluster-operator.log
+ *     │              │          └── secret
+ *     │              │              ├── builder-dockercfg-j99st.yaml
+ *     │              │              ├── default-dockercfg-pb9z6.yaml
+ *     │              │              ├── deployer-dockercfg-ltvsj.yaml
+ *     │              │              └── strimzi-cluster-operator-dockercfg-64p64.yaml
+ *     │              └── kafka-XXXXXX
+ *     │                  ├── configmap
+ *     │                  │          ├── cluster-2acba704-b-6eac3ce0-0.yaml
+ *     │                  │          ├── cluster-2acba704-bridge-config.yaml
+ *     │                  │          ├── cluster-2acba704-c-6eac3ce0-1.yaml
+ *     │                  │          ├── cluster-2acba704-entity-topic-operator-config.yaml
+ *     │                  │          ├── cluster-2acba704-entity-user-operator-config.yaml
+ *     │                  │          ├── regcred.yaml
+ *     │                  │          └── openshift-service-ca.crt.yaml
+ *     │                  ├── deployment
+ *     │                  │          ├── cluster-2acba704-bridge.yaml
+ *     │                  │          └── cluster-2acba704-entity-operator.yaml
+ *     │                  ├── events.log
+ *     │                  ├── job
+ *     │                  │          ├── admin-client-cli-create.yaml
+ *     │                  │          ├── admin-client-cli-delete.yaml
+ *     │                  │          ├── ...
+ *     │                  │          └── kafka-producer-client-zzzzzzz.yaml
+ *     │                  ├── pod
+ *     │                  │          ├── describe-pod-kroxylicious-proxy-7db664df48-bxxrx.log
+ *     │                  │          ├── ...
+ *     │                  │          └── logs-pod-kroxylicious-proxy-7db664df48-bxxrx-container-kroxylicious.log
+ *     │                  └── secret
+ *     │                      └── regcred.yaml
+ * ...
+ */
+public class TestLogCollector {
+    private static final String CURRENT_DATE;
+    private final LogCollector logCollector;
+
+    static {
+        // Get current date to create a unique folder
+        DateTimeFormatter dateTimeFormatter = DateTimeFormatter.ofPattern("yyyy-MM-dd-HH-mm-ss");
```

Comment:
I'd use https://docs.oracle.com/javase/8/docs/api/java/time/format/DateTimeFormatter.html#ISO_INSTANT rather than defining your own date/time format.

Reply:
None of the predefined formats are useful for us to define a name of a directory where the logs will be stored.

ISO_INSTANT states: "The ISO instant formatter that formats or parses an instant in UTC, such as '2011-12-03T10:15:30Z'." 

Having `:` character in a directory name seems pretty weird to me and defining our format it clearer to me than using this suggested format and then replace the characters we don't like.